heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraScheduler.java

```diff
@@ -244,6 +244,18 @@ protected String formatJavaOpts(String javaOpts) {
     auroraProperties.put(AuroraField.METRICSCACHEMGR_CLASSPATH,
         Context.metricsCacheManagerClassPath(config));
 
+    boolean isStatefulEnabled = TopologyUtils.getConfigWithDefault(
+        topology.getTopologyConfig().getKvsList(),
+        com.twitter.heron.api.Config.TOPOLOGY_STATEFUL_ENABLED, false);
+    auroraProperties.put(AuroraField.IS_STATEFUL_ENABLED, Boolean.toString(isStatefulEnabled));
+
+    String completeCkptmgrProcessClassPath = String.format("%s:%s:%s",
+        Context.ckptmgrClassPath(config),
+        Context.statefulStoragesClassPath(config),
+        Context.statefulStorageCustomClassPath(config));
+    auroraProperties.put(AuroraField.CKPTMGR_CLASSPATH, completeCkptmgrProcessClassPath);
+    auroraProperties.put(AuroraField.STATEFUL_YAML, Context.statefulFile(config));
```

Comment:
For both.

For ckptmgr, it specifies the network opts and the storage class name

For stateful storage implementation, it specifies the config based on the implementation.